tf_180.nv.json

```diff
@@ -0,0 +1,36 @@
+{
+  "_notes": [
+    "Compiled by HorsePin"
+  ],
+  "_copyright": "Copyright (C) 2022 https://github.com/horseyhorsey",
+  "_license": "GNU Lesser General Public License v3.0",
+  "_endian": "little",
+  "_ramsize": 131118,
+  "_roms": [
+    "tf_180"
+  ],
+  "_fileformat": 0.1,
+  "_version": 0.1,
+  "last_game":[
+    {"encoding":"int","length":4,"packed":false,"start":68068},
+    {"encoding":"int","length":4,"packed":false,"start":68072},
+    {"encoding":"int","length":4,"packed":false,"start":68076},
+    {"encoding":"int","length":4,"packed":false,"start":68080}
+  ],
+  "high_scores": [
+    {"label":"Autobot Grand Champion","short_label":"AGC","initials":{"start":11720,"encoding":"ch","length":3},"score":{"encoding":"int","length":4,"packed":false,"start":11744}},
+    {"label":"#1","short_label":"#2","initials":{"start":11752,"encoding":"ch","length":3},"score":{"encoding":"int","length":4,"packed":false,"start":11776}},
+    {"label":"#2","short_label":"#4","initials":{"start":11784,"encoding":"ch","length":3},"score":{"encoding":"int","length":4,"packed":false,"start":11808}},
+    {"label":"#3","short_label":"#4","initials":{"start":11816,"encoding":"ch","length":3},"score":{"encoding":"int","length":4,"packed":false,"start":11840}},
+    {"label":"#4","short_label":"#4","initials":{"start":11848,"encoding":"ch","length":3},"score":{"encoding":"int","length":4,"packed":false,"start":11872}},
+    {"label":"Decepticon Grand Champion","short_label":"AGC","initials":{"start":11880,"encoding":"ch","length":3},"score":{"encoding":"int","length":4,"packed":false,"start":11904}},
+    {"label":"#1","short_label":"#1","initials":{"start":11912,"encoding":"ch","length":3},"score":{"encoding":"int","length":4,"packed":false,"start":11936}},
+    {"label":"#2","short_label":"#2","initials":{"start":11944,"encoding":"ch","length":3},"score":{"encoding":"int","length":4,"packed":false,"start":11968}},
+    {"label":"#3","short_label":"#3","initials":{"start":11976,"encoding":"ch","length":3},"score":{"encoding":"int","length":4,"packed":false,"start":12000}},
+    {"label":"#4","short_label":"#4","initials":{"start":12008,"encoding":"ch","length":3},"score":{"encoding":"int","length":4,"packed":false,"start":12032}}
+  ],
```

Comment:
Probably need to update this with "Autobot #1" and "Deception #1" labels, and maybe "A#1" and "D#1" short labels?

Reply:
Yes. Also the deception champ is set as `AGC`